### PR TITLE
panel selection

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4954

### Change description ###
Panel hearing Function person look-up

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by rayliangatos — APPROVED** [review]
Approved, please merge after PR-1633

File: src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.scss [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+    .govuk-input {
+      width: 98% !important;
+    }
   }
-  .govuk-input {
-    width: 98% !important;
+  &.column-view{
+    .govuk-form-group {
+      .govuk-input {
+        width: 80% !important;
```
Comment (waynepHMCTS) [inline src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.scss:15]: Why are we using important

File: src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.html [hunk 2]
```diff
@@ -1,3 +1,41 @@
-<p>
-  hearing-panel works!
-</p>
+<xuilib-hmcts-error-summary [errorMessages]="validationErrors" [header]="'There is a problem'"></xuilib-hmcts-error-summary>
+
```
Comment (waynepHMCTS) [inline src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.html:2]: unnecessary space

File: src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.html [hunk 3]
```diff
@@ -1,3 +1,41 @@
...
+                 [submitButtonName]="'Exclude another person'" [isColumnView]="true"></exui-hearing-judge-names-list>
+              </div>
+            </div>
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="noSpecificPanel" name="specificPanel" (change)="showSpecificPanel('No');"
+                formControlName="specificPanel" type="radio" value="No" data-aria-controls="conditional-judge-type">
+              <label class="govuk-label govuk-radios__label" for="noSpecificPanel">No</label>
+            </div>
+            <div class="govuk-radios__conditional" [ngClass]="{'govuk-radios__conditional--hidden': panelSelection !=='No'}">
+
```
Comment (waynepHMCTS) [inline src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.html:33]: unnecessary space

File: src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.scss (hunk 1 above)
Comment (Kartheek-S) [reply src/hearings/components/hearing-judge-names-list/hearing-judge-names-list.component.scss:15]: important is used inside the lib. So, to overwrite library change used important.

File: src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.html (hunk 2 above)
Comment (Kartheek-S) [reply src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.html]: space removed

File: src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.html (hunk 3 above)
Comment (Kartheek-S) [reply src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.html]: space removed

**Review by waynepHMCTS — APPROVED** [review]

**Pull request merged**